pom.xml

```diff
@@ -10,7 +10,7 @@
     <artifactId>run-selector</artifactId>
     <packaging>hpi</packaging>
     <name>Run Selector Plugin</name>
-    <version>1.0-beta-3-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```

Comment:
I would choose 1.0.0 as the stable release version. I'd like to follow the instructions from http://semver.org/.

Opinions @ikedam @oleg-nenashev @martinda?